packages/showcase/legends/vertical-discrete-color.js

```diff
@@ -20,7 +20,7 @@
 
 import React from 'react';
 
-import DiscreteColorLegend from 'legends/discrete-color-legend';
+import DiscreteColorLegend from 'react-vis/legends/discrete-color-legend';
```

Comment:
We might want to check whether react-vis exports components in this way, since these showcase files can be used to demonstrate code and be easily-copy-pastable. But that can be a future task. This change seems to be in the right direction.

Reply:
This is the way that react-vis exports items.
Although, we should eventually change these imports only use the root `react-vis`. 
This allows us to easily curate our public api.

In general, you _shouldn't_ really import more than one level into a library as it tends to be internal.  `baseui` just recently added lint warnings if you try to go more than 2 levels deep in your imports.

Reply:
actually... I need to test this..